inst/templates/tiltWorkflows.Rmd

```diff
@@ -1,236 +1,207 @@
 ---
-title: "tilt workflow"
+title: "Creating outputs for all tilt indicators"
 author: "Kalash Singhal"
 ---
 
 ```{r, include = FALSE}
 knitr::opts_chunk$set(
   collapse = TRUE,
-  comment = "#>",
-  # FIXME when this file become reproducible
-  eval = FALSE
+  comment = "#>"
 )
 ```
 
-This file uses the output from the tiltIndicatorBefore package as the inputs to
-the tiltIndicator and tiltIndicatorAfter packages. This script uses the CSV
-files from the tiltIndicatorBefore package to create the final output of the
-four indicators.
+This workflow creates the output of all tilt indicators.
 
 ## Setup
 
 ```{r}
 library(dplyr, warn.conflicts = FALSE)
 library(readr, warn.conflicts = FALSE)
-library(tidyr, warn.conflicts = FALSE)
-# devtools::install_github("2DegreesInvesting/tiltIndicatorAfter")
-library(tiltIndicatorAfter)
```

Comment:
Calling `library(tiltWorkflows)` attaches tiltIndicatorAfter and tiltToyData automatically.